### PR TITLE
cut 0.46 release

### PR DESCRIPTION
**Pull request by s-urbaniak**
I used this little helper command to fetch the changelog entries:
```
$ for pr in $(git log --pretty="%s" --merges origin/release-0.45..HEAD | grep pull | sed -e "s/.*#\([0-9]\+\).*/\1/g" | sort -rn | uniq); do echo "PR #{$pr}:" &&  curl -s https://api.github.com/repos/prometheus-operator/prometheus-operator/pulls/${pr} 2>/dev/null | jq -r '.body' | awk '/^```release-note/,/```$/'; done
```

cc @prometheus-operator/prometheus-operator-reviewers

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.46.0 / 2021-02-24
+
+* [CHANGE] Drop support for prometheus 1.x #2822
```
Comment (simonpasquier) [inline CHANGELOG.md:3]:
```suggestion
* [CHANGE] Drop support for Prometheus 1.x #2822
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 0.46.0 / 2021-02-24
+
```
Comment (paulfantom) [inline CHANGELOG.md:2]: Can we add information that we are looking for prometheus-operator adopters entries in ADOPTERS.md file?

Comment (s-urbaniak) [reply CHANGELOG.md:2]: sure thing, you mean as a changelog entry? I am not quite clear where and how I should put that request.

**Review by simonpasquier — COMMENTED** [review]
I think we miss #3856 #3810 and #3749

File: CHANGELOG.md (hunk 2 above)
Comment (simonpasquier) [reply CHANGELOG.md:2]: not sure I would add it to the changelog. Maybe mention it when we announce the release?

Comment (paulfantom) [reply CHANGELOG.md:2]: It doesn't need to be in the changelog as this is something more for a release note.

**Comment by s-urbaniak** [comment]
hmm .. interesting ...  #3856 #3810 and #3749 don't seem to be merge commits :thinking: thanks @simonpasquier for the catch though!

File: CHANGELOG.md (hunk 1 above)
Comment (s-urbaniak) [reply CHANGELOG.md:3]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+* [CHANGE] Drop support for prometheus 1.x #2822
+* [FEATURE] Add relabelingConfigs to ProbeTargetStaticConfig #3817
+* [ENHANCEMENT] Add custom HTTP headers in remoteWrite-config #3851
```
[the remark this answers is not in the document]
Comment (s-urbaniak) [reply CHANGELOG.md:5]: fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
+## 0.46.0 / 2021-02-24
+
+* [CHANGE] Drop support for Prometheus 1.x #2822
+* [FEATURE] Add relabelingConfigs to ProbeTargetStaticConfig #3817
+* [ENHANCEMENT] Add custom HTTP headers in remoteWrite-config #3851
+* [ENHANCEMENT] CRDs are now part of prometheus-operator group allowing `kubectl get prometheus-operator` operation #3843
+* [ENHANCEMENT] Support app.kubernetes.io/managed-by label to kubelet service and endpoint objects. #3834
+* [BUGFIX] Fix the loss of EmailConfig.headers key #3856
```
Comment (simonpasquier) [inline CHANGELOG.md:8]:
```suggestion
* [BUGFIX] Fix the loss of the `headers` key in AlertmanagerConfig. #3856
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,14 @@
...
+* [BUGFIX] Preserve user added labels and annotations on Services, Endpoints and StatefulSets #3810
+* [BUGFIX] Fix child routes to not require a receiver #3749
```
Comment (simonpasquier) [inline CHANGELOG.md:10]:
```suggestion
* [BUGFIX] Do not require child routes in AlertmanagerConfig to have a receiver. #3749
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,14 @@
+## 0.46.0 / 2021-02-24
+
+* [CHANGE] Drop support for Prometheus 1.x #2822
+* [FEATURE] Add relabelingConfigs to ProbeTargetStaticConfig #3817
+* [ENHANCEMENT] Add custom HTTP headers in remoteWrite-config #3851
+* [ENHANCEMENT] CRDs are now part of prometheus-operator group allowing `kubectl get prometheus-operator` operation #3843
+* [ENHANCEMENT] Support app.kubernetes.io/managed-by label to kubelet service and endpoint objects. #3834
+* [BUGFIX] Fix the loss of EmailConfig.headers key #3856
+* [BUGFIX] Preserve user added labels and annotations on Services, Endpoints and StatefulSets #3810
```
Comment (simonpasquier) [inline CHANGELOG.md:9]:
```suggestion
* [BUGFIX] Preserve user-added labels and annotations on Service, Endpoint and StatefulSet resources managed by the operator. #3810
```

File: CHANGELOG.md (hunk 4 above)
Comment (s-urbaniak) [reply CHANGELOG.md:8]: fixed

File: CHANGELOG.md (hunk 5 above)
Comment (s-urbaniak) [reply CHANGELOG.md:10]: fixed

File: CHANGELOG.md (hunk 6 above)
Comment (s-urbaniak) [reply CHANGELOG.md:9]: fixed

**Review by simonpasquier — APPROVED** [review]
lgtm

File: CHANGELOG.md (hunk 2 above)
Comment (s-urbaniak) [reply CHANGELOG.md:2]: sure thing, I can add it in the release note section in github, where we usually copy the contents of the changelog. is that a better place?

**Pull request merged**